src/auth/auth.ts

```diff
@@ -174,7 +174,7 @@ export class Auth extends Observable<SubscriberAuth> {
     const { accessToken, error, processId } = await this.loginWithAccessOrIdTokenToken(loginOptions)
     if (!error) {
       this.accessToken = accessToken // saves in cache and in local storage
-      this.user.getData()
+      await this.user.getData()
```

Comment:
I know the method is async but in that case we don't need to wait for the return as we don't use it in the response